### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by kazalt**

### CODE REVIEW

File: src/components/TableColumnHeader.vue [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-        role="columnheader"
-        slot-scope="col"
-        :aria-sort="ariaSort"
-        :aria-disabled="ariaDisabled"
-        v-if="this.isVisible"
-    >
-        {{ label }}
-    </th>
+  <th
+    @click="clicked"
```
Comment (decaruju) [inline src/components/TableColumnHeader.vue:3]: I'd prefer an `onClick` method.

File: src/components/TableColumnHeader.vue [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+    :class="headerClass"
+    role="columnheader"
+    :aria-sort="ariaSort"
+    :aria-disabled="ariaDisabled"
+    v-if="this.isVisible"
```
Comment (decaruju) [inline src/components/TableColumnHeader.vue:8]: The order of props seems arbitrary.

Refer to CBX vue styleguide for props order

File: src/components/TableComponent.vue [hunk 3]
```diff
@@ -209,10 +213,7 @@ export default {
       let rowId = 0;
 
       this.rows = data
-        .map((rowData) => {
-          rowData.vueTableComponentInternalRowId = rowId++;
-          return rowData;
-        })
+        .map((rowData) => ({ ...rowData, vueTableComponentInternalRowId: rowId++ }))
```
Comment (decaruju) [inline src/components/TableComponent.vue]: Use map's second argument  `index` instead of rowId.

**Review by decaruju — APPROVED** [review]

File: src/components/TableColumnHeader.vue (hunk 1 above)
Comment (kazalt) [reply src/components/TableColumnHeader.vue:3]: me too, but not going to change it now

File: src/components/TableColumnHeader.vue (hunk 2 above)
Comment (kazalt) [reply src/components/TableColumnHeader.vue:8]: indeed it seems arbitrary

**Pull request merged**